docs/esa/jwst/jwst.rst

```diff
@@ -324,86 +328,66 @@ To load only table names (TAP+ capability)
 .. code-block:: python
 
   >>> from astroquery.esa.jwst import Jwst
-  >>> tables=Jwst.load_tables(only_names=True)
-  >>> for table in (tables):
+  >>> tables = Jwst.load_tables(only_names=True)
+  >>> for table in tables:
   >>>   print(table.name)
 
-  public.dual
-  tap_schema.columns
-  tap_schema.key_columns
-  tap_schema.keys
-  tap_schema.schemas
-  tap_schema.tables
-  jwst.artifact
-  jwst.chunk
-  jwst.main
-  jwst.observation
-  jwst.observationmember
-  jwst.part
-  jwst.plane
-  jwst.plane_inputs
+    ivoa.obscore
+    jwst.archive
+    jwst.artifact
+    jwst.chunk
+    jwst.harvestskipuri
+    jwst.main
+    jwst.moc
+    jwst.observation
+    ...
+
 
 To load table names (TAP compatible)
 
 .. code-block:: python
 
   >>> from astroquery.esa.jwst import Jwst
-  >>> tables=Jwst.load_tables()
-  >>> for table in (tables):
+  >>> tables = Jwst.load_tables()
+  >>> for table in tables:
   >>>   print(table.name)
 
-  public.dual
-  tap_schema.columns
-  tap_schema.key_columns
-  tap_schema.keys
-  tap_schema.schemas
-  tap_schema.tables
-  jwst.artifact
-  jwst.chunk
-  jwst.main
-  jwst.observation
-  jwst.observationmember
-  jwst.part
-  jwst.plane
-  jwst.plane_inputs
+    ivoa.obscore
+    jwst.archive
+    jwst.artifact
+    jwst.chunk
+    jwst.harvestskipuri
+    jwst.main
+    ...
 
 To load only a table (TAP+ capability)
 
 .. code-block:: python
 
   >>> from astroquery.esa.jwst import Jwst
-  >>> table=Jwst.load_table('jwst.main')
+  >>> table = Jwst.load_table('jwst.main')
   >>> print(table)
 
   TAP Table name: jwst.main
   Description:
-  Num. columns: 112
+  Num. columns: 109
 
 
 Once a table is loaded, columns can be inspected
 
 .. code-block:: python
 
   >>> from astroquery.esa.jwst import Jwst
-  >>> table=Jwst.load_table('jwst.main')
-  >>> for column in (table.columns):
-  >>>   print(column.name)
-
-  obsid
-  planeid
-  public
-  calibrationlevel
-  dataproducttype
-  algorithm_name
-  collection
-  creatorid
-  energy_bandpassname
-  ...
-  time_exposure
-  time_resolution
-  time_samplesize
-  type
-  typecode
+  >>> table = Jwst.load_table('jwst.main')
+  >>> print(*(column.name for column in table.columns), sep="\n")
```

Comment:
That line is not working for me:
AttributeError: 'TapTableMeta' object has no attribute 'colnames'

Reply:
I assumed that `table` is an instance of `astropy.table.Table`, but apparently it is an instance of `astroquery.utils.tap.model.taptable.TapTableMeta`. It is not obvious to me why the `TapTableMeta` class exists or if it has to exist then why wouldn't it inherit from `Table`, but even if that is something that should be addressed, it would be beyond the scope of this pull request.

Reply:
I agree with that, so I will leave this line as it is.